### PR TITLE
Build page with details and placeholder for logs

### PR DESCRIPTION
**Pull request by nakhbari**
Issue: #709 

```
- [x] I have run `rspec` and corrected all errors
- [x] I have run `rubocop` and corrected all errors
- [x] I have tested this change locally and tried to launch the server as well as access a project, and with that at least one build
- [x] If there is an existing issue, make sure to add `Fixes ...` as part of the PR body to reference the issue you're solving
```

#### Screenshot
image

### CODE REVIEW

File: web/app/build/build.component.html [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+      <mat-spinner *ngIf="!build" mode="indeterminate"></mat-spinner>
+      <ng-container *ngIf="build">
+        <h5>Trigger</h5>
+        <div>{{build.trigger}}</div>
+        <h5>Branch</h5>
+        <div>{{build.branch}}</div>
+        <h5>SHA</h5>
+        <div>{{build.shortSha}}</div>
+        <h5>Started</h5>
+        <div>{{build.date | amTimeAgo}}</div>
```
Comment (KrauseFx) [inline web/app/build/build.component.html]: 😍

File: web/app/models/build.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  readonly date: Date;
+
+  constructor(buildSummary: BuildResponse) {
+    this.number = buildSummary.number;
+    this.projectId = buildSummary.project_id;
+    this.description = buildSummary.description;
+    this.trigger = buildSummary.trigger;
+    this.lane = buildSummary.lane;
+    this.platform = buildSummary.platform;
+    this.parameters = buildSummary.parameters;
```
Comment (KrauseFx) [inline web/app/models/build.ts]: Looks like some changes overlap with https://github.com/fastlane/ci/pull/936, something went wrong with the branches?

Comment (nakhbari) [reply web/app/models/build.ts]: yeah this should be targeting that branch instead of master. I changed it but doesn't look like it went through.

Comment (KrauseFx) [reply web/app/models/build.ts]: 👍

**Pull request merged**